### PR TITLE
Homework 5 - logging & error handing

### PR DESCRIPTION
**Pull request by JieLiuRiver**

### CODE REVIEW

File: src/middlewares/logExceptionMiddleware.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+
+import log from '@/services/logServices';
+import { Request, Response, NextFunction } from '@/types/index';
+
+export default function logApiMiddleware(req: Request<any>, res: Response, next: NextFunction) {
+    process.on('uncaughtException', (err: Error) => {
+        console.log();
```
Comment (JonCa1) [inline src/middlewares/logExceptionMiddleware.ts]: what are these console.log(); doing here?

File: src/middlewares/logApiMiddleware.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+
+import ReqMapper from '@/mapper/reqMapper';
+import log from '@/services/logServices';
+import { Request, Response, NextFunction } from '@/types/index';
+
+
+export default function logApiMiddleware(req: Request<any>, res: Response, next: NextFunction) {
+    console.log();
```
Comment (JonCa1) [inline src/middlewares/logApiMiddleware.ts]: what are these console.log(); doing here?

File: src/middlewares/logExceptionMiddleware.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
+
+import log from '@/services/logServices';
+import { Request, Response, NextFunction } from '@/types/index';
+
+export default function logApiMiddleware(req: Request<any>, res: Response, next: NextFunction) {
+    process.on('uncaughtException', (err: Error) => {
```
Comment (JonCa1) [inline src/middlewares/logExceptionMiddleware.ts]: process.on is not needed here, as you can get the error as the first param in the error handling middleware callback

File: src/middlewares/logApiMiddleware.ts (hunk 2 above)
Comment (JieLiuRiver) [reply src/middlewares/logApiMiddleware.ts]: I want to add a blank before outputting accurate info, so I can check it easier.

File: src/middlewares/logExceptionMiddleware.ts (hunk 3 above)
Comment (JieLiuRiver) [reply src/middlewares/logExceptionMiddleware.ts]: Thank you let me know that

File: src/middlewares/logExceptionMiddleware.ts (hunk 1 above)
Comment (JieLiuRiver) [reply src/middlewares/logExceptionMiddleware.ts]: I just want to add a blank before outputting accurate info, so I can check it easier.

Comment (JonCa1) [reply src/middlewares/logExceptionMiddleware.ts]: this is not needed

File: src/middlewares/logApiMiddleware.ts (hunk 2 above)
Comment (JonCa1) [reply src/middlewares/logApiMiddleware.ts]: this is not needed

**Comment by JonCa1** [comment]
I have just checked the next homework PR, https://github.com/JieLiuRiver/nodejs-learning-project/pull/6/files#diff-f542a3cb3e5114a42087046cde21f8b8e7de05d59ec717b53752d1c01d453c14, the accurate code changes is not reflected, please clear up the changes to reflect the accurate code changes in this PR, then ping me for reviewing, thanks!

**Pull request merged**